.travis.yml

```diff
@@ -51,9 +49,10 @@ before_script:
 
 # Install composer dependencies
   - composer validate
-  - composer require silverstripe/recipe-testing:^1 silverstripe/installer:4.6.x-dev --no-update
-  - if [[ $DB == PGSQL ]]; then composer require silverstripe/postgresql:2.x-dev --no-update; fi
-  - if [[ $DB == SQLITE ]]; then composer require silverstripe/sqlite3:2.x-dev --no-update; fi
```

Comment:
Was removing SQLite here deliberate?

Reply:
Yes, it was not in the matrix so it's unused